EssentialEstablishmentGenerator/Tavern/js/createTavern.js

```diff
@@ -76,8 +76,24 @@ setup.createTavern = function (town, opts) {
   tavern.lodging = ''
   tavern.sin = ''
   tavern.food = ''
+  tavern.colour1 = [setup.colours.yellow.colour.seededrandom(), setup.colours.orange.colour.seededrandom(), setup.colours.red.colour.seededrandom(), setup.colours.purple.colour.seededrandom(), setup.colours.blue.colour.seededrandom(), setup.colours.green.colour.seededrandom(), setup.colours.brown.colour.seededrandom(), setup.colours.black.colour.seededrandom(), setup.colours.white.colour.seededrandom()].seededrandom()
```

Comment:
haven't finished the colour function; this is fine for now, but the functionality will allow for specifying "paintable" colours, or an array. So this is more a comment for my benefit, really.